### PR TITLE
fix: website accessibility fixes

### PR DESCRIPTION
**Pull request by hippee-lee**
- update image labels for app-layout images
- fixes for icon buttons
- properly label getting started intro for supported browsers image

Signed-off-by: Matt Hippely <mhippely@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?
Adds more accessible behavior to website content.

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Adds more accessible behavior to website content.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: apps/website/get-started/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Clarity’s designs stem from continuous exploration and research. These designs
 
 ## Device and Browser Support
 
-<ClrImage src="/images/get-started/device_support.png" width="100%" />
+<ClrImage src="/images/get-started/device_support.png" width="100%" aria-label="Image showing supported browser list that container icons for: IE11, Edge, Chrome, Safari and Firefox."/>
```
Comment (mathisscott) [inline apps/website/get-started/README.md:14]: I think this is supposed to say `that contains icons for: ` instead of `that container icons for:`

**Review by mathisscott — CHANGES_REQUESTED** [review]

**Review by mathisscott — APPROVED** [review]

**Comment by bbogdanov** [comment]
@hippee-lee I have some code for solving other issues, so would you like to push my code into your branch and get all the fixes we have at once?

**Comment by bbogdanov** [comment]
Seems also like the file we had with the issue is not much of a great way to track what we do because we did a double work here

**Comment by hippee-lee** [comment]
> @hippee-lee I have some code for solving other issues, so would you like to push my code into your branch and get all the fixes we have at once?

Let's keep the changes separate. If the code is changes in a similar way it will just rebase overtop but if they are different we can discuss that on the new PR.

Sorry about the confusion - perhaps the spreadsheet is not updating?

**Review by bbogdanov — APPROVED** [review]

**Pull request merged**